assets/js/src/core/modules/asset/editor/editor-container.tsx

```diff
@@ -38,7 +39,7 @@ const EditorContainer = (props: EditorContainerProps): React.JSX.Element => {
 
   useEffect(() => {
     if (isWidgetActive) {
-      setContext({ id })
+      setContext({ id, context: getBaseAssetContext(asset) })
```

Comment:
It feels a bit strange that we have a "context" in the `context` - maybe we should name this somehow different - e.g. "contextIdentifiers" or similar?